jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/redhatpolicies/misc/RedhatNotScanningClusterPolicy.java

```diff
@@ -0,0 +1,100 @@
+        }
+        String policyId = policyParam.get(PacmanSdkConstants.POLICY_ID);
+        String severity = policyParam.get(PacmanRuleConstants.SEVERITY);
+        String category = policyParam.get(PacmanRuleConstants.CATEGORY);
+        MDC.put("executionId", policyParam.get("executionId"));
+        MDC.put("policyId", policyId);
+        try {
+            String pacmanHost = PacmanUtils.getPacmanHost(PacmanRuleConstants.ES_URI);
+            String clusterName = resourceAttributes.get("name");
+            String region = resourceAttributes.get("region");
+            String accountId = resourceAttributes.get(PacmanRuleConstants.ACCOUNTID);
+            Map<String, Object> mustFilter = new HashMap<>();
+            mustFilter.put("status.providerMetadata.google.clusterName", clusterName);
+            mustFilter.put("status.providerMetadata.zone", region);
+            mustFilter.put("status.providerMetadata.google.project", accountId);
+            HashMultimap<String, Object> shouldFilter = HashMultimap.create();
+            Map<String, Object> mustTermsFilter = new HashMap<>();
+            String endPoint = pacmanHost + REDHAT_CLUSTER_URL;
+            Set<String> resourceName = PacmanUtils.getValueFromElasticSearchAsSet(endPoint, mustFilter, shouldFilter,
+                    mustTermsFilter, "_resourcename", null);
+            if (resourceName == null || resourceName.isEmpty()) {
+                List<LinkedHashMap<String, Object>> issueList = new ArrayList<>();
+                LinkedHashMap<String, Object> issue = new LinkedHashMap<>();
+                Annotation annotation = Annotation.buildAnnotation(policyParam, Annotation.Type.ISSUE);
+                annotation.put(PacmanSdkConstants.DESCRIPTION, "Cluster - " + clusterName + " not scanned by redhat acs found");
+                annotation.put(PacmanRuleConstants.SEVERITY, severity);
+                annotation.put(PacmanRuleConstants.CATEGORY, category);
+                issue.put(PacmanRuleConstants.VIOLATION_REASON, "Cluster - " + clusterName + " not scanned by redhat acs found");
+                issueList.add(issue);
+                annotation.put("issueDetails", issueList.toString());
+                logger.debug("========RedhatNotScanningClusterPolicy ended with annotation {} : =========", annotation);
+                return new PolicyResult(PacmanSdkConstants.STATUS_FAILURE, PacmanRuleConstants.FAILURE_MESSAGE, annotation);
+            }
+        } catch (Exception e) {
+            logger.error("Unable to scan cluster", e);
+            throw new RuleExecutionFailedExeption(e.getMessage());
+        }
+        logger.debug("========RedhatNotScanningClusterPolicy ended=========");
+        return new PolicyResult(PacmanSdkConstants.STATUS_SUCCESS, PacmanRuleConstants.SUCCESS_MESSAGE);
+    }
+
+    @Override
+    public String getHelpText() {
+        return "Checks for the presence of vulnerabilities.";
```

Comment:
Are we using this anywhere? if yes, we may need to update this. @akashpaladin

Reply:
hi @santhosh-challa I have updated the help text, and I haven't seen this method used